packages/plugins/CyberConnect/src/base.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { NetworkPluginID } from '@masknet/plugin-infra'
+import { CYBERCONNECT_PLUGIN_ID } from './constants'
+import { ChainId } from '@masknet/web3-shared-evm'
+import { CyberConnectIcon } from './assets/CyberConnectIcon'
+export const base: Plugin.Shared.Definition = {
+    ID: CYBERCONNECT_PLUGIN_ID,
+    icon: <CyberConnectIcon />,
+    name: { fallback: 'CyberConnect' },
+    description: {
+        fallback: 'A plugin for https://cyberconnect.me/',
+    },
+    publisher: { name: { fallback: 'CyberConnect' }, link: 'https://github.com/cyberconnecthq' },
```

Comment:
Can you represent CyberConnect? If not, change this field to yourself 🤔

Reply:
yeah I'm team member of CyberConnect